models/discordactions.js

```diff
@@ -831,6 +840,175 @@ const updateUsersWith31DaysPlusOnboarding = async () => {
   }
 };
 
+const getMissedProgressUpdatesUsers = async (options = {}) => {
```

Comment:
The function is quite lengthy and it can be divided into smaller functions in order to enhance its understandability.